examples/asset-search/README.md

```diff
@@ -10,14 +10,23 @@
 
 ### How to start locally
 
-- Run `npm install` to install dependencies.
-- Run `npm start` to start developing. \
-  Your URL should be similar to this example:
-  ```
-  http://localhost:3000
-  ```
-- Paste the URL under **App URL** in your [app settings](https://developers.miro.com/docs/build-your-first-hello-world-app#step-2-create-your-app-in-miro).
-- Open a board; you should see your app in the app toolbar or in the **Apps** panel.
+1. Run `npm install` to install dependencies.
+2. Run `npm start` to start developing. \
+   Your URL should be similar to this example:
+   ```
+   http://localhost:3000
+   ```
+3. Open the [app manifest editor](https://developers.miro.com/docs/manually-create-an-app#step-2-configure-your-app-in-miro) by clicking **Edit in Manifest**. \
+   In the app manifest editor, configure the app as follows:
+
```

Comment:
@mettin yes this is correct: https://developers.miro.com/docs/build-your-first-hello-world-app#step-1-bootstrap-the-hello-world-app

This part of the README is here in case they want to either review/check or change the preconfigured settings.

Suggestion:

* I can add here a link to the docs where we show how the [app install with automatic link to app manifest config](https://developers.miro.com/docs/build-your-first-hello-world-app#step-1-bootstrap-the-hello-world-app) works. \
Do you think it would be helpful?

Reply:
No I dont think thats needed. Im just thinking that we could generate the links and put them in the readme files on build somehow. For a follow up